.circleci/config.yml

```diff
@@ -319,3 +366,37 @@ workflows:
               ignore: /.*/
           requires:
             - hold
+  build_and_deploy_search_bar:
+    jobs:
+      - build_search_bar:
+          filters:
+            tags:
+              only: /^search-bar-v.*/
+            branches:
+              ignore: /.*/
+      - headless_acceptance_test_search_bar:
+          filters:
+            tags:
+              only: /^search-bar-v.*/
```

Comment:
I forget why, but we're not able to just have the first step have the `filters: tags: only: /^search-bar` option right, with the other steps automatically getting the filter?

Reply:
Filters to don't automatically pass through the same way branches do. You need to include filters at every step, but branches will automatically pass through